### DISCUSSION TITLE
What data is shared in a dump file?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [winsiderss/systeminformer#1965] [Questions / Answers] [answered]
I've been having crashes in two machines which happen constantly for some time then the crashes stop for weeks, then start again (this all between updates) but I'm hesitant to share the dump files because I don't know what data is stored in them. Can you please tell me what data they contain and how to read it? It would be great to have a log file where I can redact private info from it (as I do in other software), can I edit the dump files too? I don't even know how to read them, Windows doesn't have a reader built-in, does it?

**Comment by jxy-s** [comment, marked as answer]
It is not clear to me from your message if this is a question about System Informer or not. Generically any dump file can contain sensitive information, usually dependent on where the memory originated from and how much information was captured. Kernel mode dumps are likely to contain information from across the entire system, some of which you may not want shared with others. It depends on what you do with the machine. If it is a user mode dump it is likely to only contain information pertinent to that process' address space. Again you may or may not want to share that information depending on the nature of the program or your operating system - and what you do with on/with that system.

If this is a dump from a System Informer process it will contain information that System Informer can display to you already. Which includes broader information about your system. If you have a dump file caused by System Informer and you do not wish to share it publicly, you can welcome to email the maintainers the dump file directly.

With respect to reading the dump files yourself. Use WinDbg: https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/
